packages/dynamoose/lib/utils/dynamoose/returnModel.ts

```diff
@@ -9,7 +9,7 @@ export default <T extends Item = AnyItem>(model: Model<T>): ModelType<T> => {
 		Object.keys(model),
 		Object.keys(Object.getPrototypeOf(model)),
 		Object.getOwnPropertyNames(Object.getPrototypeOf(model))
-	]).filter((key) => !["constructor", "name"].includes(key));
+	]).filter((key) => !["constructor", "name", "tableName"].includes(key));
```

Comment:
What is the purpose of this change?

Reply:
Due to moving tableName property from ModelInternalProperties to root of Model class, it must be filtered here just like the "name" property